mongodbatlas.go

```diff
@@ -126,6 +128,56 @@ func (m *MongoDBAtlas) NewUser(ctx context.Context, req dbplugin.NewUserRequest)
 		return dbplugin.NewUserResponse{}, err
 	}
 
+	clusters, _, err := client.Clusters.List(ctx, m.ProjectID, &mongodbatlas.ListOptions{})
```

Comment:
this api call is paginated, if someone has more than 100 clusters you may miss those outside the [1,100] range

Reply:
It seems this is not correctly implemented on the client, will raise an issue with the mongo team